pycm/pycm_obj.py

```diff
@@ -218,6 +218,26 @@ def __iter__(self):
         for key in self.matrix.keys():
             yield key, self.matrix[key]
 
+    def __contains__(self, class_name):
+        """
+        Check if the confusion matrix contains given class name.
+
+        :param class_name: given class name
+        :param class_name: any valid type
+        :return: True if class_name is a class in confusion matrix
+        """
+        return class_name in self.classes
+
+    def __getitem__(self, class_name):
+        """
+        Return value in matrix corresponding to the given class name.
+
+        :param class_name: given class name
+        :type class_name: any valid type
+        :return: value in matrix corresponding to the given class name
```

Comment:
It's better to say:
":return: the element(s) in matrix ..."

Reply:
fixed in https://github.com/sepandhaghighi/pycm/pull/476/commits/d8bb025060a64d0e778ee7fd0691b94fd9d56974